### PR TITLE
Bounce auto

### PR DESCRIPTION
**Pull request by YJ-7274**

### CODE REVIEW

File: src/main/java/frc/robot/commands/BounceAuto.java [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+import edu.wpi.first.wpilibj.trajectory.constraint.DifferentialDriveVoltageConstraint;
+import edu.wpi.first.wpilibj2.command.RamseteCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Constants;
+import frc.robot.FieldMapHome;
+import frc.robot.subsystems.DriveTrain;
+public class BounceAuto extends SequentialCommandGroup implements AutoCommandInterface {
+  Rotation2d rotation180 = Rotation2d.fromDegrees(180.0);
+  Rotation2d rotation270 = Rotation2d.fromDegrees(270.0);
+  Rotation2d rotation90 = Rotation2d.fromDegrees(90.0);
```
Comment (prensing) [inline src/main/java/frc/robot/commands/BounceAuto.java:23]: these 3 should probably be "final".

Comment (prensing) [reply src/main/java/frc/robot/commands/BounceAuto.java:23]: Also, the indent of these lines is wrong.

**Review by prensing — CHANGES_REQUESTED** [review]
Please clean up the formatting a bit: add some blank lines (eg before the constructor), delete extra ones in a few places.

File: src/main/java/frc/robot/commands/BounceAuto.java (hunk 1 above)
Comment (YJ-7274) [reply src/main/java/frc/robot/commands/BounceAuto.java]: I fixed those changes.

**Review by coachFitz0807 — APPROVED** [review]

**Review by prensing — APPROVED** [review]

**Pull request merged**